onnxruntime/core/mlas/lib/sqnbitgemm_kernel_neon.cpp

```diff
@@ -1175,7 +1375,6 @@
     }
 }
 
-MLAS_FORCEINLINE
```

Comment:
why remove this?

Reply:
it didn't make sense here. this function pointer is assigned to this function.

https://github.com/microsoft/onnxruntime/blob/da1f68385a0b980e3f9a09a53fe5be731a924726/onnxruntime/core/mlas/lib/sqnbitgemm_kernel_neon.cpp#L1434